tabs/led_strip.html

```diff
@@ -123,11 +115,15 @@
                     <input type="checkbox" name="Indicator" class="toggle function-i" />
                     <label> <span>Indicator (uses position on matrix)</span></label>
                 </div>
+                <div class="checkbox vtxOverlay">
+                    <input type="checkbox" name="Vtx" class="toggle function-v" />
+                    <label> <span>VTX (uses vtx frequency to assign color)</span></label>
```

Comment:
Can you please put GUI text into https://github.com/betaflight/betaflight-configurator/blob/master/_locales/en/messages.json , to make translation easier? They then can be used like this: https://github.com/sheaivey/betaflight-configurator/blob/dbd2fdace67858430f71f06b64d7b2ef16b1d052/tabs/led_strip.html#L3